contracts/DappCampNFT.sol

```diff
@@ -5,26 +5,121 @@ import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 import "@openzeppelin/contracts/utils/Strings.sol";
 import "@openzeppelin/contracts/utils/Base64.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
-
+import "hardhat/console.sol";
```

Comment:
Always remove this before deployment since an extra contract is added here and that increases gas significantly

Reply:
@shubham-kanodia I've tried multiple deployments to allow me to mint more than 1 NFT, but keep receiving the same error message. Do you know why I cannot mint more than 1 NFT in my collection?
<img width="464" alt="image" src="https://user-images.githubusercontent.com/11972548/169349268-61c26680-cb05-4594-93bd-135ec607dd4f.png">

Reply:
This maybe due to issues with rinkeby network. I am facing the same issue locally with a collection that I was able to deploy on opensea earlier